### PR TITLE
Add testing

### PR DESCRIPTION
**Pull request by brucellino**
I've added a default scenario for testing with molecule. No tests have been added.
Also added a conditional to address #2 

Addresses #1

### CODE REVIEW

**Review by enolfc — CHANGES_REQUESTED** [review]
Could you rebase this on top of #4 once merged?

**Comment by brucellino** [comment]
I _think_ I've rebased... waiting for checks to finish.

File: templates/voms.json.j2 [hunk 1]
```diff
@@ -1,8 +1,10 @@
 {
+{% if caso_vos %}
```
Comment (enolfc) [inline templates/voms.json.j2:2]: if you rebased these changes should not appear here :(

**Review by enolfc — COMMENTED** [review]
Let me try to rebase your changes and re-send the PR...

File: templates/voms.json.j2 (hunk 1 above)
Comment (brucellino) [reply templates/voms.json.j2:2]: I thought that was your change... Sorry man, rebases are my kryptonite.

**Comment by enolfc** [comment]
Moving this to #5, to have a nicer history

**Pull request closed without merging**